<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>removeTopic</class>
 <widget class="QDialog" name="removeTopic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Topic</string>
  </property>
  <property name="windowIcon">
   <iconset resource="feedIOicons.qrc">
    <normaloff>:/images/feedIO.png</normaloff>:/images/feedIO.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="topicListCombo">
       <property name="minimumSize">
        <size>
         <width>291</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="whatsThis">
          <string/>
         </property>
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="feedIOicons.qrc"/>
 </resources>
 <connections/>
</ui>
